sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -500,6 +502,17 @@ public OutT apply(KafkaRecord<KeyT, ValueT> record) {
             return new KafkaConsumer<>(config);
           }
         };
+
+    @SuppressWarnings("unchecked")
+    @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
+      super.populateDisplayData(builder);
+      builder.add(DisplayData.item("topics", Joiner.on(",").join(getTopics())));
+      for (Map.Entry<String, Object> conf : getConsumerConfig().entrySet()) {
```

Comment:
So, the consumer properties map is what the user passed to the `Read` transform builder. 
It is then passed to Kafka consumer which joins these properties with Kafka defaults, but this is in a different scope.
So this map won't contain the defaults for each configuration Kafka has but rather just the ones the user specified in `KafkaIO.Read#updateConsumerProperties`